examples/reference/widgets/Toggle.ipynb

```diff
@@ -16,37 +16,36 @@
    "cell_type": "markdown",
```

Comment:
`Toggle.variant` is not working

![image](https://github.com/user-attachments/assets/b0e276b6-b065-4704-b78a-07e5ab4ffb95)

```python
import panel_material_ui as pmui

pmui.FlexBox(
    pmui.Toggle(label="contained", variant="contained", value=True),
    pmui.Toggle(label="outlined", variant="outlined", value=True),
    pmui.Toggle(label="text", variant="text", value=True),
).servable()
```

Reply:
Doesn't actually seem to be supported by Mui. Will refactor to drop the parameter.

Reply:
Removed.